List.java

```diff
@@ -0,0 +1,15 @@
+
+import java.util.ArrayList;
+
+public class List {
+	 public static void main(String[] args) {
+		 ArrayList<String> list = new ArrayList<>();
```

Comment:
Listインターフェースを扱うときは変数の型を実装クラスであるArrayListではなくListにしましょう！
```suggestion
		 List<String> list = new ArrayList<>();
```
理由は多態性（ポリモーフィズム）が関係します。
が、難しいのでひとまず書き方を覚えるでOKです！